migrations/versions/d763cf2e7dc4_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: d763cf2e7dc4
+Revises: 1c6f7e989021
+Create Date: 2017-04-23 12:44:43.830776
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'd763cf2e7dc4'
+down_revision = '1c6f7e989021'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(u'events_creator_id_fkey', 'events', type_='foreignkey')
+    op.drop_column('events', 'creator_id')
```

Comment:
The issue initially was that if the `creator_id` column was not there at the first place, why was it being dropped in the migration. But it seems like `creator_id` field was deleted from the model but no migration file was generated for the same. This issue is resolved.